codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -37,6 +37,28 @@
     void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream, FlowControlled payload);
 
     /**
+     * Flush all writes queued in this flow controller. This allows the flow controller to coalesce smaller
+     * individual writes into a single frame to reduce framing overhead and improve efficiency of receiver.
+     *
+     * @throws Http2Exception throws if a protocol-related error occurred.
+     */
+    void flushPendingBytes() throws Http2Exception;
+
+    /**
+     * Add a listener on flow-control write events.
+     *
+     * @param listener to notify when the a write occurs.
+     */
+    void addListener(WriteListener listener);
```

Comment:
But in that case the class name is `Http2Connection.Listener`, not `ConnectionListener` ... so `addListener` makes sense.  I'd just like to be consistent between the method and interface names.  I'm fine with renaming `WriteListener` to just `Listener`.